### ISSUE TITLE
Flaky tests: RazorPagesTemplate_NoAuth_Works_NetCore et al

### ISSUE THREAD

**Issue by ryanbrandenburg** [aspnet/Templating#621]
This test [fails](http://aspnetci/viewLog.html?buildId=507950&buildTypeId=Lite_UniverseTest) occasionally with the following error:
```
Expected file to exist, but it doesn't: AspNet.Template.49019daf5e2b412c9188d2cf81a843f1.csproj
Expected: True
Actual:   False
   at Templates.Test.TemplateTestBase.ReadFile(String path) in /_/test/Templates.Test/Helpers/TemplateTestBase.cs:line 199
   at Templates.Test.RazorPagesTemplateTest.RazorPagesTemplate_NoAuthImpl(String targetFrameworkOverride, Boolean noHttps) in /_/test/Templates.Test/RazorPagesTemplateTest.cs:line 34
------- Stdout: -------
==> C:\b\w\33bdfc1cae7b2a38\.dotnet\x64\dotnet.exe new razor --debug:custom-hive "C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp3.0\.templateengine" [C:\b\w\33bdfc1cae7b2a38\modules\Templating\test\Templates.Test\bin\Release\netcoreapp3.0\TestTemplates\AspNet.Template.49019daf5e2b412c9188d2cf81a843f1]
The template "Razor Page" was created successfully.
```

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=514334&buildTypeId=Lite_UniverseTest).

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=514430&buildTypeId=Lite_UniverseTest).

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515134&buildTypeId=Lite_UniverseTest).

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=514929&buildTypeId=Lite_UniverseTest).

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515383&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515336&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515544&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515782&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515977&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=515925&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516180&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516660&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516645&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516549&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516714&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516700&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516834&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516957&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=516943&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=517017&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=517196&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=517757&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=517599&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518163&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518436&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518450&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518510&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518585&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518699&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=518926&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519073&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519091&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519450&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519339&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519558&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519492&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519586&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519572&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519624&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed again](http://aspnetci/viewLog.html?buildId=519689&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed with about the same error](http://aspnetci/viewLog.html?buildId=520895&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed with about the same error](http://aspnetci/viewLog.html?buildId=522188&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed with about the same error](http://aspnetci/viewLog.html?buildId=523211&buildTypeId=Releases_22xPublic_Win2012).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed with about the same error](http://aspnetci/viewLog.html?buildId=524711&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
RazorPagesTemplate_NoAuth_Works_NetCore [failed with about the same error](http://aspnetci/viewLog.html?buildId=524676&buildTypeId=Lite_UniverseTest).

This comment was made automatically. If there is a problem contact @ryanbrandenburg.

**Comment by ryanbrandenburg** [comment]
Judging by the frequency with which this failed before I think this was fixed by my recent blast of fixes to Templating. I'm not exactly sure what I did to fix it though. @ajaybhargavb I'll leave it to you to decide if you want to dig more or call it fixed.

**Comment by ajaybhargavb** [comment]
Fixed by @ryanbrandenburg with d95f9716932cae55933c0a4700fec3513c55c92f
